src/Facade/Services/SalesArticleForecastService.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Linn.Products.Facade.Services
+{
+    using System;
+
+    using Linn.Common.Facade;
+    using Linn.Common.Persistence;
+    using Linn.Products.Domain.Linnapps.Products;
+    using Linn.Products.Resources;
+
+    public class SalesArticleForecastService : FacadeService<SalesArticle, string, SalesArticleResource>
+    {
+        public SalesArticleForecastService(
+            IRepository<SalesArticle, string> repository,
+            ITransactionManager transactionManager)
+            : base(repository, transactionManager)
+        {
+        }
+
+        protected override SalesArticle CreateFromResource(SalesArticleResource resource)
+        {
+            throw new NotImplementedException();
```

Comment:
is this because you inherit from FacadeService rather than just making a new service definition than has just the methods you need in this case the update

Reply:
yeah. I'll never have to create one of these. 

Could split the facade service into two with a create version inheriting an update only version. 
Didn't seem worth it really but I'm easy.